spec/yaml/tsg_mc_contacts_v3.yaml

```diff
@@ -642,26 +732,51 @@ paths:
       summary: Import Contacts
       tags:
       - Contacts
-      description: "**This endpoint allows a CSV upload containing up to one million\
-        \ contacts or 5GB of data, whichever is smaller.**\n\nImports take place asynchronously:\
-        \ the endpoint returns a URL (`upload_uri`) and HTTP headers (`upload_headers`)\
-        \ which can subsequently be used to `PUT` a file of contacts to be  imported\
-        \ into our system.\n\nUploaded CSV files may also be [gzip-compressed](https://en.wikipedia.org/wiki/Gzip).\n\
-        \nIn either case, you must include the field `file_type` with the value `csv`\
-        \ in your request body.\n\nThe `field_mappings` paramter is a respective list\
-        \ of field definition IDs to map the uploaded CSV columns to. It allows you\
-        \ to use CSVs where one or more columns are skipped (`null`) or remapped to\
-        \ the contact field. \n\nFor example, if `field_mappings` is set to `[null,\
-        \ \"w1\", \"_rf1\"]`, this means skip column 0, map column 1 to the custom\
-        \ field with the ID `w1`, and map column 2 to the reserved field with the\
-        \ ID `_rf1`. See the \"Get All Field Definitions\" endpoint to fetch your\
-        \ custom and reserved field IDs to use with `field_mappings`.\n\nOnce you\
-        \ recieve the response body you can then initiate a **second** API call where\
-        \ you use the supplied URL and HTTP header to upload your file. For example:\n\
-        \n`curl --upload-file \"file/path.csv\" \"URL_GIVEN\" -H 'HEADER_GIVEN'`\n\
-        \nIf you'd like to monitor the status of your import job, use the `job_id`\
-        \ and the \"Import Contacts Status\" endpoint.\n\nTwilio SendGrid recommends\
-        \ exporting your contacts regularly as a backup to avoid issues or lost data."
+      description: '**This endpoint allows a CSV upload containing up to one million
+        contacts or 5GB of data, whichever is smaller. At least one identifier is
+        required for a successful import.**
+
+
+        Imports take place asynchronously: the endpoint returns a URL (`upload_uri`)
+        and HTTP headers (`upload_headers`) which can subsequently be used to `PUT`
+        a file of contacts to be  imported into our system.
+
```

Comment:
Just making sure the double spacing was intentional. I see in other places we go with \n instead.